tasks/sysroot.py

```diff
@@ -87,9 +85,21 @@ def install(c):
 ]
 
 
-@task(platforms="linux", archs="armv7l")
+@task(platforms="linux", archs="armv7l,aarch64")
 def install(c):
 
+    if c.arch == "armv7l":
+        deb_arch = "armhf"
+        deb_mirror = "http://archive.raspbian.org/raspbian"
+        qemu_arch = "arm"
+    elif c.arch == "aarch64":
+        deb_arch = "arm64"
+        deb_mirror = "http://deb.debian.org/debian"
+        qemu_arch = "aarch64"
+
+    c.var("deb_arch", deb_arch)
+    c.var("qemu_arch", qemu_arch)
```

Comment:
I just realized that I didn't add a line for `deb_mirror` here, but when I tested it, it worked fine. What's the purpose of these calls supposed to be then?